File: CarroNetPhp/locadoras/VehicleCharges.php
<?php

/**
 * VehicleCharges short summary.
 *
 * VehicleCharges description.
 *
 * @version 1.0
 * @author Eduardo Assis
 */
class VehicleCharges
{
    private $vehicleChargeIncludedInRate;
    private $vehicleChargeRateConvertInd;
    private $vehicleChargeIncludedInEstTotalInd;
    private $vehicleChargeDescription;
    private $vehicleChargeDecimalPlaces;
    private $vehicleChargeCurrencyCode;
    private $vehicleChargeTaxInclusive;
    private $vehicleChargeAmount;
    private $vehicleChargePurpose;
    private $calculationUnitCharge;
    private $calculationUnitName;
    private $calculationQuantity;
    private $calculationTotal;

    public function getVehicleChargeIncludedInRate()
    {
        return $this->vehicleChargeIncludedInRate;
    }

    public function setVehicleChargeIncludedInRate($value)
    {
        $this->vehicleChargeIncludedInRate = $value;
        return $this;
    }

    public function getVehicleChargeRateConvertInd()
    {
        return $this->vehicleChargeRateConvertInd;
    }

    public function setVehicleChargeRateConvertInd($value)
    {
        $this->vehicleChargeRateConvertInd = $value;
        return $this;
    }

    public function getVehicleChargeIncludedInEstTotalInd()
    {
        return $this->vehicleChargeIncludedInEstTotalInd;
    }

    public function setVehicleChargeIncludedInEstTotalInd($value)
    {
        $this->vehicleChargeIncludedInEstTotalInd = $value;
        return $this;
    }

    public function getVehicleChargeDescription()
    {
        return $this->vehicleChargeDescription;
    }

    public function setVehicleChargeDescription($value)
    {
        $this->vehicleChargeDescription = $value;
        return $this;
    }

    public function getVehicleChargeDecimalPlaces()
    {
        return $this->vehicleChargeDecimalPlaces;
    }

    public function setVehicleChargeDecimalPlaces($value)
    {
        $this->vehicleChargeDecimalPlaces = $value;
        return $this;
    }

    public function getVehicleChargeCurrencyCode()
    {
        return $this->vehicleChargeCurrencyCode;
    }

    public function setVehicleChargeCurrencyCode($value)
    {
        $this->vehicleChargeCurrencyCode = $value;
        return $this;
    }

    public function getVehicleChargeTaxInclusive()
    {
        return $this->vehicleChargeTaxInclusive;
    }

    public function setVehicleChargeTaxInclusive($value)
    {
        $this->vehicleChargeTaxInclusive = $value;
        return $this;
    }

    public function getVehicleChargeAmount()
    {
        return $this->vehicleChargeAmount;
    }

    public function setVehicleChargeAmount($value)
    {
        $this->vehicleChargeAmount = $value;
        return $this;
    }

    public function getVehicleChargePurpose()
    {
        return $this->vehicleChargePurpose;
    }

    public function setVehicleChargePurpose($value)
    {
        $this->vehicleChargePurpose = $value;
        return $this;
    }

    public function getCalculationUnitCharge()
    {
        return $this->calculationUnitCharge;
    }

    public function setCalculationUnitCharge($value)
    {
        $this->calculationUnitCharge = $value;
        return $this;
    }

    public function getCalculationUnitName()
    {
        return $this->calculationUnitName;
    }

    public function setCalculationUnitName($value)
    {
        $this->calculationUnitName = $value;
        return $this;
    }

    public function getCalculationQuantity()
    {
        return $this->calculationQuantity;
    }

    public function setCalculationQuantity($value)
    {
        $this->calculationQuantity = $value;
        return $this;
    }

    public function getCalculationTotal()
    {
        return $this->calculationTotal;
    }

    public function setCalculationTotal($value)
    {
        $this->calculationTotal = $value;
        return $this;
    }
}

File: Carronet/Models/VehicleCharges.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Carronet.Models
{
    public class VehicleCharges
    {
        string vehicleCharge_IncludedInRate;
        string vehicleCharge_RateConvertInd;
        string vehicleCharge_IncludedInEstTotalInd;
        string vehicleCharge_Description;
        string vehicleCharge_DecimalPlaces;
        string vehicleCharge_CurrencyCode;
        string vehicleCharge_TaxInclusive;
        string vehicleCharge_Amount;
        string vehicleCharge_Purpose;
        string calculation_UnitCharge;
        string calculation_UnitName;
        string calculation_Quantity;
        string calculation_Total;

        public string VehicleCharge_IncludedInRate { get => vehicleCharge_IncludedInRate; set => vehicleCharge_IncludedInRate = value; }
        public string VehicleCharge_RateConvertInd { get => vehicleCharge_RateConvertInd; set => vehicleCharge_RateConvertInd = value; }
        public string VehicleCharge_IncludedInEstTotalInd { get => vehicleCharge_IncludedInEstTotalInd; set => vehicleCharge_IncludedInEstTotalInd = value; }
        public string VehicleCharge_Description { get => vehicleCharge_Description; set => vehicleCharge_Description = value; }
        public string VehicleCharge_DecimalPlaces { get => vehicleCharge_DecimalPlaces; set => vehicleCharge_DecimalPlaces = value; }
        public string VehicleCharge_CurrencyCode { get => vehicleCharge_CurrencyCode; set => vehicleCharge_CurrencyCode = value; }
        public string VehicleCharge_TaxInclusive { get => vehicleCharge_TaxInclusive; set => vehicleCharge_TaxInclusive = value; }
        public string VehicleCharge_Amount { get => vehicleCharge_Amount; set => vehicleCharge_Amount = value; }
        public string VehicleCharge_Purpose { get => vehicleCharge_Purpose; set => vehicleCharge_Purpose = value; }
        public string Calculation_UnitCharge { get => calculation_UnitCharge; set => calculation_UnitCharge = value; }
        public string Calculation_UnitName { get => calculation_UnitName; set => calculation_UnitName = value; }
        public string Calculation_Quantity { get => calculation_Quantity; set => calculation_Quantity = value; }
        public string Calculation_Total { get => calculation_Total; set => calculation_Total = value; }
    }
}